.github/workflows/drivers.yml

```diff
@@ -346,3 +346,22 @@ jobs:
       uses: ./.github/actions/test-driver
       with:
         junit-name: 'be-tests-sqlserver-ee'
+
+  be-tests-snowflake-ee:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 120
```

Comment:
can you make a follow up issue that in a month we look at the historical duration of this new test? I'd like to bump this down as soon as we have a baseline for how long it should run. 2 hours seems wild. If in a month we find these tests do take 2 hours, we should fix that for sure.

Reply:
Likely because of issue 26054, so it shouldn't need special timeout from now on.

Reply:
I think we fixed problems with Snowflake that caused it to run that long. At one point we had to bump the timeout in CircleCI to 115 minutes.

We'll closely monitor all driver tests. As @camsaul noted in another PR, all driver tests seem to be getting slower and slower. Maybe we should investigate that once we migrate them all to GitHub Actions.

Reply:
Actually, I had to rebase to solve the merge conflicts anyway so I dialed down this to the 60 mintues. cc @dpsutton